feat(gpu): add troubleshooting index (#4585)
scaleway/docs-content: pages/gpu/troubleshooting/index.mdx

---
meta:
  title: GPU Instances - Troubleshooting
  description: Find troubleshooting resources for resolving issues with GPU Instances.
content:
  paragraph: Find troubleshooting resources for resolving issues with GPU Instances.
dates:
  posted: 2025-03-10
categories:
  - gpu-instances
---

<ProductHeader
  productName="GPU Instances troubleshooting"
  productLogo="instances"

>   description="Learn how to fix common issues with Scaleway GPU Instances. Find solutions to help resolve problems and get started."

Reviewer: ```suggestion
  description="Learn how to fix common issues with Scaleway GPU Instances."
```

/>

## Featured Pages

<Grid>
    <DefaultCard
      title="Instances FAQ"

>       description="General info on Scaleway Instances"

Author: ```suggestion
      description="General info on Scaleway Instances."
```

      url="/faq/instances/"
      label="See more"
    />
    <DefaultCard
      title="Choosing the right GPU Instance type"
      description="Learn how to choose the right GPU Instance for your workloads."
      url="/gpu/reference-content/choosing-gpu-instance-type/"
      label="See more"
    />
    <DefaultCard
      title="GPU Instances internet and Block Storage bandwidth overview"

>       description="Inform yourself about the internet and Block Storage bandwidth available for each GPU Instance type."

Reviewer: ```suggestion
      description="Learn about the internet and Block Storage bandwidth available for each GPU Instance type."
```

>       description="GPU workloads often involve processing large datasets, requiring high-bandwidth networking and storage access."

Author: ```suggestion
      description="Inform yourself about the internet and Block Storage bandwidth available for each GPU Instance type."
```

      url="/gpu/reference-content/gpu-instances-bandwidth-overview/"
      label="See more"
    />
</Grid>
<Grid>
    <DefaultCard
      title="NVIDIA GPU time-slicing with Kubernetes"

>       description="Learn how to use time-slicing by sharing a physical GPU among multiple containers or pods in a Kubernetes cluster."

Reviewer: ```suggestion
      description="Learn to share a physical GPU among multiple containers or pods in a Kubernetes cluster."
```

      url="/gpu/reference-content/kubernetes-gpu-time-slicing/"
      label="See more"
    />
    <DefaultCard
      title="Installing NVIDIA drivers on Ubuntu 22.04"

>       description="Learn how to manually install the NVIDIA drivers on an Ubuntu 22.04 image, adapted from the default OS image."

Reviewer: ```suggestion
      description="Learn to manually install the NVIDIA drivers on an Ubuntu 22.04 image."
```
